pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio_test.go

```diff
@@ -362,44 +339,24 @@ func TestResourceBinPackingSingleExtended(t *testing.T) {
 }
 
 func TestResourceBinPackingMultipleExtended(t *testing.T) {
-	extendedResource1 := "intel.com/foo"
-	extendedResource2 := "intel.com/bar"
 	extendedResources1 := map[string]int64{
 		"intel.com/foo": 4,
 		"intel.com/bar": 8,
 	}
-
 	extendedResources2 := map[string]int64{
 		"intel.com/foo": 8,
 		"intel.com/bar": 4,
 	}
 
-	extnededResourcePod1 := v1.PodSpec{
-		Containers: []v1.Container{
-			{
-				Resources: v1.ResourceRequirements{
-					Requests: v1.ResourceList{
-						v1.ResourceName(extendedResource1): resource.MustParse("2"),
-						v1.ResourceName(extendedResource2): resource.MustParse("2"),
-					},
-				},
-			},
-		},
+	extendedResourcePod1 := map[v1.ResourceName]string{
```

Comment:
The current code uses `extendedResourcePod1` since these are pod resources so I think keeping the name as it is makes more sense.

Reply:
But it's a bad naming I think. `extendedResourcePod1` indicates it's a pod not the pod resource. We used to name this   `extendedResourcePod1` because it's actually a pod.

Reply:
Sure, updated.